### PR TITLE
Fix: Fixed highlights flickering after closing the properties window

### LINKED ISSUE #11461: Bug: Highlights are flickering after closing the properties window
### Description

As described in the title.
https://user-images.githubusercontent.com/66369541/221338552-d4b394f9-8e3a-41b3-9e46-55159c05079d.mp4

### Steps To Reproduce

1. Open a properties window for some item.
2. Close the properties window.
3. The highlight flickers when hovering over an item.

### Steps to fix

Avoid flickering highlights.

### Files Version

2.4.44.0

### Windows Version

10.0.22621.1265

### Log file

[debug.log](https://github.com/files-community/Files/files/10830274/debug.log)

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
Items resolved / related issues by this PR.
- Closes #11461 

**Validation**
How did you test these changes?
- [X] Built and ran the app
- [X] Tested the changes for accessibility

### CODE REVIEW

File: src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs [hunk 1]
```diff
@@ -308,7 +309,7 @@ private void Page_KeyDown(object sender, KeyRoutedEventArgs e)
 		private void ClosePage()
 		{
 			if (_usingWinUI)
-				AppWindow.Destroy();
+				Window.Close(); // AppWindow.Destroy() doesn't seem to work well. (#11461)
```
Comment (yaira2) [inline src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs:312]: Can you move the comment above the if statement?

Comment (yaira2) [reply src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs:312]: Could be under the if statement, main thing is to put it on top of Window.Close..

Comment (hishitetsu) [reply src/Files.App/Views/Properties/MainPropertiesPage.xaml.cs]: OK. Moved.

**Review by yaira2 — APPROVED** [review]
LGTM

**Pull request merged**